README.md

```diff
@@ -166,6 +166,26 @@ The various path options are only needed if Tiger can't find the paths on its ow
 * `--help` Print help.
 * `--version` Print version
 
+## Benchmarks
```

Comment:
I'm thinking this doesn't fit the audience of the main README, which is aimed at users.

Maybe it could be placed in benches/ ? Though then directing people to benches/ is a bit useless.

Maybe we need a `CODING.md`